hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientService.java

```diff
@@ -88,12 +93,23 @@ public SqlClientService(HazelcastClientInstanceImpl client) {
         long resubmissionTimeoutMillis = client.getProperties().getPositiveMillisOrDefault(INVOCATION_TIMEOUT_SECONDS);
         this.resubmissionTimeoutNano = TimeUnit.MILLISECONDS.toNanos(resubmissionTimeoutMillis);
         this.resubmissionRetryPauseMillis = client.getProperties().getPositiveMillisOrDefault(INVOCATION_RETRY_PAUSE_MILLIS);
+
+        this.isSmartRouting = client.getClientConfig().getNetworkConfig().isSmartRouting();
```

Comment:
Since we now have Alto support, we need to use the method in the connection manager (isUnisocketClient)